docs/getting-started/agentless-troubleshooting.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[agentless-integration-troubleshooting]]
+= Agentless integrations FAQ
+
+Frequently asked questions and troubleshooting steps for {elastic-sec}'s agentless CSPM integration.
+
+*When I make a new integration, how long until the agent appears on the agent policies page?*
```

Comment:
When a new agentless integration is created, then the agent count should appear on Integrations Policies Page, not agent policies page.